### ISSUE TITLE
trinity service: cannot start on trinity nodes

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#372]
**Version:** r5_fixes f3f13d1
**Environment:** amaster

```
[root@node001 ~]# systemctl status trinity.service -l
trinity.service - LSB: start and stop trinity
   Loaded: loaded (/etc/rc.d/init.d/trinity)
   Active: failed (Result: exit-code) since Wed 2015-07-29 14:18:19 CEST; 13s ago
  Process: 2818 ExecStart=/etc/rc.d/init.d/trinity start (code=exited, status=1/FAILURE)

Jul 29 14:18:12 node001.cluster trinity[2818]: Cannot find device "docker0"
Jul 29 14:18:12 node001.cluster trinity[2818]: bridge docker0 doesn't exist; can't delete it
Jul 29 14:18:12 node001.cluster trinity[2818]: Host c001.vc-0 not found: 3(NXDOMAIN)
Jul 29 14:18:12 node001.cluster trinity[2818]: c001.vc-a has address 172.16.0.1
Jul 29 14:18:16 node001.cluster trinity[2818]: Error response from daemon: Cannot start container trinity: error gathering device information while adding custom device "/dev/infiniband": not a device node
Jul 29 14:18:16 node001.cluster trinity[2818]: time="2015-07-29T14:18:16+02:00" level=fatal msg="Error: failed to start one or more containers"
Jul 29 14:18:19 node001.cluster trinity[2818]: Docker inspect returned invalid PID 0
Jul 29 14:18:19 node001.cluster systemd[1]: trinity.service: control process exited, code=exited status=1
Jul 29 14:18:19 node001.cluster systemd[1]: Failed to start LSB: start and stop trinity.
Jul 29 14:18:19 node001.cluster systemd[1]: Unit trinity.service entered failed state.
```

**Comment by hansthen** [comment]
Fixed in 107a07d81501d8061a1b902cefcdac6061a40225

**Comment by themisa** [comment]
Fixed in **Version:** r5_fixes e5f4623
